### DISCUSSION TITLE
World Package 1.1.2버전이 업데이트 되었습니다.

### DISCUSSION THREAD

**Discussion by hyeonjin-jo** [naverz/zepeto-studio-kor#186] [1. Announcements]
안녕하세요.

World Package 1.1.2버전이 업데이트되어서 안내드립니다.

*릴리즈노트 링크 : https://studio.zepeto.me/kr/guides/world-112

제페토앱 3.9.0이상부터 호환되는 것 참고 부탁드립니다.

감사합니다.

**Comment by Deytooh** [comment]
Hi 안녕하세요 ~ 수고하셨습니다 ! , I am trying to use Zepeto World with Unity to develop worlds for Zepeto, but the documentation is really not enough. Are you planning to expend the current documentation soon ?
